lib/broccoli/ember-app.js

```diff
@@ -38,6 +38,8 @@ var merge         = require('lodash/object/merge');
 var omit          = require('lodash/object/omit');
 var Funnel        = require('broccoli-funnel');
 
+var deprecate     = require('../utilities/deprecate');
```

Comment:
Oh, actually... those `ui` helpers don't exist on `stable-1-13`. This will have to go through `deprecateUI`, or those helpers will have to be pulled in, too. (#5179)